spec/display-buffer-spec.coffee

```diff
@@ -703,6 +715,11 @@ describe "DisplayBuffer", ->
       expect(displayBuffer.screenPositionForBufferPosition([100000, 0])).toEqual [12, 2]
       expect(displayBuffer.screenPositionForBufferPosition([100000, 100000])).toEqual [12, 2]
 
+    it "clips correctly around atomic tab characters", ->
+      buffer.insert([0, 0], '\t')
```

Comment:
Moreover, what do you think to handle also every column between 0 and 4 in this test?